frontend/packages/core-api-client/src/lib/types/client/Recipient.ts

```diff
@@ -0,0 +1,8 @@
+import {FieldValue, Record, RecordList} from '../model/Record';
+import {RecordListRequest, RecordGetRequest} from "./Record";
+
+export type RecipientDefinition = FieldValue[]
```

Comment:
I think a `RecipientDefinition` is actually just `Omit<Recipient, 'id'>`. Technically the backend will accept an id, but I think that's bad.